packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1269,6 +1270,11 @@ export type ParticipantsRemovedListener = (event: {
     removedBy: ChatParticipant;
 }) => void;
 
+// @beta
+export interface RemoteVideoTileMenuOptions {
+    isHidden?: boolean;
```

Comment:
is there a reason we have isHidden optional as `remoteVideoTileMenu ` is already optional, so in case it is passed it should have `isHidden` value

Reply:
We chose to have a prop `isHidden` in an object to allow for future options to `remoteVideoTileMenu` for the CallComposite. See this discussion in this PR: https://github.com/Azure/communication-ui-library/pull/2611#discussion_r1052444394